greenlight/lib/api/keys.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, marionette_getter):
 
     @property
     def ACCEL(self):
-        if self.marionette_getter().session_capabilities['platformName'] == 'DARWIN':
+        if self.marionette_getter().session_capabilities['platformName'] == \
+                'DARWIN':
```

Comment:
Which linter are you using? I have `Python Flake8 Lint` installed in Sublime Edit and it doesn't blame anything.

I will rewrite this code above given that it doesn't make sense to do this check all the time. It's enough to do in __init__.